settings/default/main.lua

```diff
@@ -111,26 +111,27 @@ xi.settings.main =
     DARK_POWER      = 1.000, -- Multiplies amount drained by Dark Magic.
     ITEM_POWER      = 1.000, -- Multiplies the effect of items such as Potions and Ethers.
     WEAPON_SKILL_POWER  = 1.000, -- Multiplies damage dealt by Weapon Skills.
+
     USE_ADOULIN_WEAPON_SKILL_CHANGES = true, -- true/false. Change to toggle new Adoulin weapon skill damage calculations
+    DISABLE_PARTY_EXP_PENALTY        = false, -- true/false.
 
     -- TRUSTS
     ENABLE_TRUST_CASTING           = 1,
     ENABLE_TRUST_QUESTS            = 1,
     ENABLE_TRUST_CUSTOM_ENGAGEMENT = 0,
-    ENABLE_TRUST_CASTING = 1,
-    ENABLE_TRUST_QUESTS  = 1,
-    ENABLE_TRUST_ALTER_EGO_EXTRAVAGANZA = 0, -- 0 = disabled, 1 = summer/ny, 2 = spring/autumn, 3 = both
+
+    ENABLE_TRUST_ALTER_EGO_EXTRAVAGANZA          = 0, -- 0 = disabled, 1 = summer/ny, 2 = spring/autumn, 3 = both
     ENABLE_TRUST_ALTER_EGO_EXTRAVAGANZA_ANNOUNCE = 0, -- 0 = disabled, 1 = add announcement to player login
-    ENABLE_TRUST_ALTER_EGO_EXPO = 0, -- 0 = disabled, 1 = expo - HPP/MPP/Status Resistance, 2 = expo plus (not implemented)
-    ENABLE_TRUST_ALTER_EGO_EXPO_ANNOUNCE = 0, -- 0 = disabled, 1 = add announcement to player login
+    ENABLE_TRUST_ALTER_EGO_EXPO                  = 0, -- 0 = disabled, 1 = expo - HPP/MPP/Status Resistance, 2 = expo plus (not implemented)
+    ENABLE_TRUST_ALTER_EGO_EXPO_ANNOUNCE         = 0, -- 0 = disabled, 1 = add announcement to player login
 
     TRUST_ALTER_EGO_EXTRAVAGANZA_MESSAGE =
-        "\n \n" ..
+        "\n\n" ..
```

Comment:
The extra space was intentional. \n\n doesn't generate a blank line as expected.

\n\n
![image](https://user-images.githubusercontent.com/3996176/181244141-e5c98eff-80e9-462b-ab22-fa3023bd09be.png)

\n \n
![image](https://user-images.githubusercontent.com/3996176/181244395-a405c863-5e6c-4366-8340-8155982c810a.png)

Reply:
Strange, alrighty I'll change it back 👍

Reply:
I suspect that funky SE multibyte encoding that plagues the server login message struck here